front/src/modules/simulationResult/components/TrainDetails.tsx

```diff
@@ -30,33 +31,48 @@ const getOccupancyBounds = (
 };
 
 export default function TrainDetails() {
-  const { positionValues, timePosition, consolidatedSimulation, selectedTrainId } = useSelector(
+  const { consolidatedSimulation, selectedTrainId } = useSelector(
     (state: RootState) => state.osrdsimulation
   );
 
+  const [localTimePosition, setLocalTimePosition] = useState<Date>(new Date());
+  const [localPositionValues, setLocalPositionValues] = useState<PositionsSpeedTimes<Date>>(
+    {} as PositionsSpeedTimes<Date>
+  );
+  useChartSynchronizer(
+    (timePosition, positionValues) => {
+      setLocalTimePosition(timePosition);
+      setLocalPositionValues(positionValues);
+    },
+    'train-detail',
+    []
+  );
+
   const { t } = useTranslation(['simulation']);
 
   const occupancyBounds = useMemo(() => {
     const foundTrain = consolidatedSimulation.find((train) => train.id === selectedTrainId);
 
     return (
       foundTrain &&
-      getOccupancyBounds(foundTrain.routeAspects, new Date(positionValues?.headPosition?.time))
+      getOccupancyBounds(foundTrain.routeAspects, new Date(localPositionValues?.headPosition?.time))
     );
-  }, [consolidatedSimulation, positionValues, selectedTrainId]);
+  }, [consolidatedSimulation, localPositionValues, selectedTrainId]);
 
   return (
     <div className="d-flex">
-      {positionValues.headPosition && timePosition && (
+      {localPositionValues?.headPosition && localTimePosition && (
```

Comment:
Remove this condition, render only the relevant divs
```suggestion
```

Reply:
this component is just weird, I think I’ll do that in a new PR